torch/_dynamo/eval_frame.py

```diff
@@ -813,6 +886,10 @@ def graph_with_interpreter(*args):
         graph,
     ).transform()
 
+    new_graph = AddRuntimeAssertsInInputConstraint(
```

Comment:
This is not the full set of runtime assertions. Anything we specialized on technically should be checked as well.

Reply:
Yeah this is only for Input constraints. I will make similar change for inline constraints as well